torch/csrc/jit/backends/backend_debug_handler.cpp

```diff
@@ -34,5 +36,23 @@ std::unordered_map<int64_t, DebugInfoPair> BackendDebugHandleManager::
   return handles_to_inlined_callstack_ptrs_;
 }
 
+WithBackendDebugInfoRecorder::WithBackendDebugInfoRecorder(
```

Comment:
Since we hvae not updated preprocess interface we still need thread_local in order for generate_debug_handles to pick up the BackendDebugInfoRecorder.